From 7b53f69784a5b2f0730dc48da4d5441ecba48d29 Mon Sep 17 00:00:00 2001
From: harvey186 <harvey186@hotmail.com>
Date: Mon, 14 Jul 2025 18:05:20 +0200
Subject: [PATCH] ROOT-mozConfig

---

 configs/common/mozconfig | 24 +++++++-----------
 2 files changed, 64 insertions(+), 15 deletions(-)



diff --git a/configs/common/mozconfig b/configs/common/mozconfig
index 6aadab7..aaded2d 100644
--- a/configs/common/mozconfig
+++ b/configs/common/mozconfig
@@ -1,16 +1,16 @@
 # Browser branding
 ac_add_options --with-app-name=${binName}
-ac_add_options --with-app-basename=Zen
+ac_add_options --with-app-basename=Snake
 
 # Localization (Must be an absolute path)
 ac_add_options --with-l10n-base="${topsrcdir}/browser/locales"
 
 export MOZ_USER_DIR="${name}"
-export MOZ_APP_BASENAME=Zen
+export MOZ_APP_BASENAME=Snake
 export MOZ_BRANDING_DIRECTORY=${brandingDir}
 export MOZ_OFFICIAL_BRANDING_DIRECTORY=${brandingDir}
 
-ac_add_options --with-distribution-id=app.zen-browser
+ac_add_options --with-distribution-id=app.snake-browser
 
 # Uncomment if builds are too resource hungry
 # mk_add_options MOZ_MAKE_FLAGS="-j4"
@@ -32,12 +32,6 @@ if ! test "$SCCACHE_GHA_ENABLED" = "false"; then
   fi
 fi
 
-# add safe browsing key if it exists on a file
-if test -f "$HOME/.zen-keys/safebrowsing.dat"; then
-  ac_add_options --with-google-safebrowsing-api-keyfile="$HOME/.zen-keys/safebrowsing.dat"
-fi
-
-if test "$ZEN_RELEASE"; then
 
   # TODO: Make this successful in builds
   # ac_add_options --enable-clang-plugin
@@ -97,7 +91,7 @@ if test "$ZEN_RELEASE"; then
   ac_add_options --enable-updater
 
   export MOZ_PACKAGE_JSSHELL=1
-fi
+
 
 ac_add_options --enable-unverified-updates
 
@@ -105,12 +99,12 @@ ac_add_options --enable-jxl
 
 ac_add_options --with-unsigned-addon-scopes=app,system
 
-mk_add_options MOZ_DATA_REPORTING=
-mk_add_options MOZ_SERVICES_HEALTHREPORT=
-mk_add_options MOZ_TELEMETRY_REPORTING=
+#mk_add_options MOZ_DATA_REPORTING=
+#mk_add_options MOZ_SERVICES_HEALTHREPORT=
+#mk_add_options MOZ_TELEMETRY_REPORTING=
 
-ac_add_options MOZ_DATA_REPORTING=
-ac_add_options MOZ_TELEMETRY_REPORTING=
+#ac_add_options MOZ_DATA_REPORTING=
+#ac_add_options MOZ_TELEMETRY_REPORTING=
 
 # Allow loading unsigned extensions
 export MOZ_REQUIRE_SIGNING=
-- 
2.50.1

