From 9aed9b8cae45e505eeaa215d0d87e42d50bb3eca Mon Sep 17 00:00:00 2001
From: harvey186 <harvey186@hotmail.com>
Date: Sat, 12 Jul 2025 18:22:58 +0200
Subject: [PATCH] Telemetry_1

---
 modules/libpref/init/all.js | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/modules/libpref/init/all.js b/modules/libpref/init/all.js
index c292e75aa2..f62f023a2b 100644
--- a/modules/libpref/init/all.js
+++ b/modules/libpref/init/all.js
@@ -543,6 +543,11 @@ pref("toolkit.sqlitejsm.loglevel", "Error");
 
 pref("toolkit.tabbox.switchByScrolling", false);
 
+pref("network.trr.confirmation_telemetry_enabled", false);
+pref("identity.fxaccounts.telemetry.clientAssociationPing.enabled", false);
+pref("browser.search.serpEventTelemetryCategorization.enabled", false);
+
+
 // Telemetry settings.
 // Server to submit telemetry pings to.
 //pref("toolkit.telemetry.server", "https://incoming.telemetry.mozilla.org");
-- 
2.50.1

