diff --git a/modules/libpref/init/all.js b/modules/libpref/init/all.js
index 95bcd71793..dfecc258ea 100644
--- a/modules/libpref/init/all.js
+++ b/modules/libpref/init/all.js
@@ -3264,8 +3264,8 @@ pref("network.connectivity-service.enabled", true);
 pref("network.connectivity-service.DNSv4.domain", "example.org");
 pref("network.connectivity-service.DNSv6.domain", "example.org");
 pref("network.connectivity-service.DNS_HTTPS.domain", "cloudflare-dns.com");
-pref("network.connectivity-service.IPv4.url", "http://detectportal.firefox.com/success.txt?ipv4");
-pref("network.connectivity-service.IPv6.url", "http://detectportal.firefox.com/success.txt?ipv6");
+pref("network.connectivity-service.IPv4.url", "http://captiveportal.kuketz.de/generate_204");
+pref("network.connectivity-service.IPv6.url", "http://captiveportal.kuketz.de/generate_204");
 
 pref("network.trr.uri", "");
 // credentials to pass to DOH end-point
@@ -3278,8 +3278,8 @@ pref("network.trr.confirmationNS", "example.com");
 pref("network.trr.excluded-domains", "");
 pref("network.trr.builtin-excluded-domains", "localhost,local");
 
-pref("captivedetect.canonicalURL", "http://detectportal.firefox.com/canonical.html");
-pref("captivedetect.canonicalContent", "<meta http-equiv=\"refresh\" content=\"0;url=https://support.mozilla.org/kb/captive-portal\"/>");
+pref("captivedetect.canonicalURL", "http://captiveportal.kuketz.de/generate_204");
+//pref("captivedetect.canonicalContent", "<meta http-equiv=\"refresh\" content=\"0;url=https://support.mozilla.org/kb/captive-portal\"/>");
 pref("captivedetect.maxWaitingTime", 5000);
 pref("captivedetect.pollingTime", 3000);
 pref("captivedetect.maxRetryCount", 5);

diff --git a/netwerk/docs/captive_portals.md b/netwerk/docs/captive_portals.md
index 6606088b54..c7ea46709b 100644
--- a/netwerk/docs/captive_portals.md
+++ b/netwerk/docs/captive_portals.md
@@ -48,7 +48,7 @@ pref("network.captive-portal-service.maxInterval", 1500000); // 25 minutes
 pref("network.captive-portal-service.backoffFactor", "5.0");
 
 // The URL used to perform the captive portal checks
-pref("captivedetect.canonicalURL", "http://detectportal.firefox.com/canonical.html");
+//pref("captivedetect.canonicalURL", "http://captiveportal.kuketz.de");
 // The response we expect to receive back for the canonical URL
 // It contains valid HTML that when loaded in a browser redirects the user
 // to a support page explaining captive portals.
